ibis/expr/types/strings.py

```diff
@@ -58,25 +58,25 @@ def __getitem__(self, key: slice | int | ir.IntegerScalar) -> StringValue:
         │ r                     │
         └───────────────────────┘
         >>> t.food[:3]
-        ┏━━━━━━━━━━━━━━━━━━━━━━━┓
-        ┃ Substring(food, 0, 3) ┃
-        ┡━━━━━━━━━━━━━━━━━━━━━━━┩
-        │ string                │
-        ├───────────────────────┤
-        │ bre                   │
-        │ che                   │
-        │ ric                   │
-        └───────────────────────┘
+        ┏━━━━━━━━━━━━━━━━━━━━━━┓
+        ┃ StringSlice(food, 3) ┃
```

Comment:
this repr isn't great: both [:3] and [3:] result in `StringSlice(food, 3)`. But that is separate problem.